src/routes/mainRouter.js

```diff
@@ -10,4 +17,12 @@ router.post("/signup", signupDataValidate, signUp);
 router.route("/login").post(loginDataValidate, logIn).get(authenticateUser);
 router.get("/logout", logOut);
 
+//notes
+
+router.post("/addnote", authenticateUser, addNote);
```

Comment:
I recommend using a route naming convention that was suggested by Elijah in a slack message ([link](https://codethedream.slack.com/archives/C061FVBGZU0/p1699964703056139))

Reply:
This is how I see the routes:
`/user/ID/notes`, get
`/note/ID/`, update
`/note/ID/`, delete
`/user/ID/notes`, delete
where ID is userId or noteId depending on a situation

Reply:
renamed routes